grpc/android-cubic/sdk/src/main/java/cobaltspeech/cubic/CubicGrpc.java

```diff
@@ -0,0 +1,518 @@
+package cobaltspeech.cubic;
+
+import static io.grpc.MethodDescriptor.generateFullMethodName;
+import static io.grpc.stub.ClientCalls.asyncBidiStreamingCall;
+import static io.grpc.stub.ClientCalls.asyncClientStreamingCall;
+import static io.grpc.stub.ClientCalls.asyncServerStreamingCall;
+import static io.grpc.stub.ClientCalls.asyncUnaryCall;
+import static io.grpc.stub.ClientCalls.blockingServerStreamingCall;
+import static io.grpc.stub.ClientCalls.blockingUnaryCall;
+import static io.grpc.stub.ClientCalls.futureUnaryCall;
+import static io.grpc.stub.ServerCalls.asyncBidiStreamingCall;
+import static io.grpc.stub.ServerCalls.asyncClientStreamingCall;
+import static io.grpc.stub.ServerCalls.asyncServerStreamingCall;
+import static io.grpc.stub.ServerCalls.asyncUnaryCall;
+import static io.grpc.stub.ServerCalls.asyncUnimplementedStreamingCall;
+import static io.grpc.stub.ServerCalls.asyncUnimplementedUnaryCall;
+
+/**
+ * <pre>
+ * Service that implements the Cobalt Cubic Speech Recognition API
+ * </pre>
+ */
+@javax.annotation.Generated(
+    value = "by gRPC proto compiler (version 1.24.0)",
+    comments = "Source: cubic.proto")
+public final class CubicGrpc {
+
+  private CubicGrpc() {}
+
+  public static final String SERVICE_NAME = "cobaltspeech.cubic.Cubic";
+
+  // Static method descriptors that strictly reflect the proto.
+  private static volatile io.grpc.MethodDescriptor<com.google.protobuf.Empty,
+      cobaltspeech.cubic.CubicOuterClass.VersionResponse> getVersionMethod;
+
+  @io.grpc.stub.annotations.RpcMethod(
```

Comment:
It looks like there are some indentation issues with the method below

Reply:
It is generated code